.github/workflows/main-build.yml

```diff
@@ -153,12 +153,13 @@ jobs:
         query: ^workflows
   
     - name: Build nextflow schemas
-      uses: viash-io/viash-actions/pro/build-nextflow-schemas@v5
+      uses: viash-io/viash-actions/pro/build-nextflow-schemas@v5.1.0
```

Comment:
This can be kept at `v5`
```suggestion
      uses: viash-io/viash-actions/pro/build-nextflow-schemas@v5
```
Because `v5` is a sliding tag (meaning we update it to be the latest release in the `0.5.x` series)

Reply:
ah perfect!